Abstract Update (#75)
thomas-fossati/draft-tls13-iot: draft-ietf-uta-tls13-iot-profile.md

--- abstract

> RFC 7925 offers guidance to developers on using TLS/DTLS 1.2 for Internet of
> Things (IoT) devices with resource constraints. This document serves as a
> companion to RFC 7925, defining TLS/DTLS 1.3 profiles for IoT devices.
> Additionally, it updates RFC 7925 with respect to the X.509 certificate
> profile and ciphersuite requirements.

Reviewer: ```suggestion
RFC 7925 offers guidance to developers on using TLS/DTLS 1.2 for Internet of
Things (IoT) devices with resource constraints. This document is a
companion to RFC 7925, defining TLS/DTLS 1.3 profiles for IoT devices.
Additionally, it updates RFC 7925 with respect to the X.509 certificate
profile and ciphersuite requirements.
```
